tensorflow/python/eager/forwardprop.py

```diff
@@ -439,3 +439,7 @@ def _fetch_jvp(tensor):
         return array_ops.zeros_like(tensor)
       return result
     return nest.map_structure(_fetch_jvp, primals)
+
+  @classmethod
+  def _batch_accumulator(cls, primals, unconnected_gradients=UnconnectedGradients.NONE):
```

Comment:
I think using .jvp for fetching batch jvps is fine once batching is enabled, it's the constructor argument I'm concerned about. Can we take out the constructor argument and instead construct a batching ForwardAccumulator with a classmethod for now?

The issue is whether there's a public API for creating/using the batching mode. I don't think there should be until we know it composes with other common transformations.